src/pl_bolts/models/gans/dcgan/dcgan_module.py

```diff
@@ -200,6 +200,16 @@ def cli_main(args=None):
         )
         dataset = MNIST(root=script_args.data_dir, download=True, transform=transforms)
         image_channels = 1
+    elif script_args.dataset == "cifar10":
+        transforms = transform_lib.Compose(
+            [
+                transform_lib.Resize(script_args.image_size),
+                transform_lib.ToTensor(),
+                transform_lib.Normalize((0.5, 0.5, 0.5), (0.5, 0.5, 0.5)),
```

Comment:
is this right with respect to the dataset color distribution?

Reply:
the official PyTorch example seem to use the same normalization - https://pytorch.org/tutorials/beginner/blitz/cifar10_tutorial.html